cmd/podman/generate_systemd.go

```diff
@@ -40,7 +39,10 @@ func init() {
 	containerSystemdCommand.SetHelpTemplate(HelpTemplate())
 	containerSystemdCommand.SetUsageTemplate(UsageTemplate())
 	flags := containerSystemdCommand.Flags()
-	flags.BoolVarP(&containerSystemdCommand.Name, "name", "n", false, "use the container name instead of ID")
+	flags.BoolVarP(&containerSystemdCommand.Name, "name", "n", false, "use the container/pod name instead of ID")
+	if !remoteclient {
+		flags.BoolVarP(&containerSystemdCommand.Files, "files", "f", false, "generate files instead of printing on stdout")
```

Comment:
nit, and only a suggestion:  s/on stdout/to stdout/

Reply:
Good one, thanks!